test/cmd.js

```diff
@@ -56,23 +56,23 @@ test('check if stdout goes to output', t => {
   const result = shell.cmd('shx', 'echo', 'this is stdout');
   t.falsy(shell.error());
   t.is(result.code, 0);
-  t.is(result.stdout, 'this is stdout\n');
+  t.is(result.stdout, 'this is stdout');
 });
 
 test('check if stderr goes to output', t => {
   const result = shell.cmd(shell.config.execPath, '-e', 'console.error("1234");');
   t.falsy(shell.error());
   t.is(result.code, 0);
   t.is(result.stdout, '');
-  t.is(result.stderr, '1234\n');
+  t.is(result.stderr, '1234');
```

Comment:
Can we leave the newline at the end?

I think this option will fix it: [stripFinalNewline](https://github.com/sindresorhus/execa/tree/v5.0.0?tab=readme-ov-file#stripfinalnewline)

Reply:
sure can!